lumberjack.go

```diff
@@ -403,6 +408,11 @@ func (l *Logger) millRunOnce() error {
 	return err
 }
 
+func shouldCompressFile(compressAll bool, firstFile bool, filename string) bool {
+	alreadyCompressed := strings.HasSuffix(filename, compressSuffix)
+	return (compressAll && !alreadyCompressed) || (!compressAll && !firstFile && !alreadyCompressed)
```

Comment:
Personal preference - for me the following is more readable than starting to calculate the logic in my head:
```go
if alreadyCompressed {
	return false
}
return compressAll || !firstFile
```

Reply:
This changed due to the previous change, but I simplified as you suggested.